.github/workflows/ci.yml

```diff
@@ -27,6 +27,20 @@ jobs:
       - run: pip install black
       - run: black --check --diff modin/ asv_bench/benchmarks scripts/doc_checker.py
 
+  lint-mypy:
+    name: lint (mypy)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8.x"
+          architecture: "x64"
+      - run: pip install -r requirements-dev.txt
```

Comment:
Can we just `pip install mypy` as we do with `black` above?

Reply:
I decided to install all of it because I think `mypy` might require all packages to be installed so that it can do the type checking.